contracts/staking/AutoRewardsDistributor.sol

```diff
@@ -0,0 +1,60 @@
+pragma solidity ^0.8.0;
+
+import "./TribalChief.sol";
```

Comment:
Add a TribalChief interface and use that

Reply:
Done